CoreWordCloudView/release/modules/ext/docs/CoreWordCloudView/src/au/gov/asd/tac/constellation/views/wordcloud/word-cloud-view.md

```diff
@@ -33,6 +33,44 @@ comparison.
 <img src="../ext/docs/CoreWordCloudView/src/au/gov/asd/tac/constellation/views/wordcloud/resources/WordCloudView.png" alt="Word Cloud View" />
 
 </div>
+<br />
 
 
+### Element Type
 
+The type of element to gather data from when generating the word cloud. Currently only supports Nodes nad Transactions.
```

Comment:
could these parameters be formatted to be more like what you see for parameters in the Data Access plugins? (Perhaps a title of Generate Word Cloud rather than Parameters)